src/components/sponsors/style.module.css

```diff
@@ -40,12 +40,6 @@
 			opacity: 1;
 		}
 
-		:global(.dark) &:hover img,
-		:global(.dark) &:hover svg {
-			filter: invert(0) hue-rotate(0deg) grayscale(0)
-				drop-shadow(0 0 0.5px #fff) drop-shadow(0 5px 10px #000)
-				drop-shadow(0 0 20px rgba(255, 255, 255, 0.5));
-		}
```

Comment:
We don't use `.dark` (before my time, might've been a preference switcher that was removed at some point?), but this causes issues as it requires `:is` which is not available in the default Chrome version (87 is the default, needs 88. So close!).

What the targets should be is another topic but figured we can at least drop these unused styles that breach the defaults for now.

Reply:
Ah yeah, I think I removed the switcher at some point to make room in the header.